lib/fetch/util.js

```diff
@@ -178,7 +178,11 @@ function toWebReadable (streamReadable) {
     if (err) {
       controller.error(err)
     } else {
-      controller.close()
+      // Must not use `process.nextTick()`,
+      // see https://github.com/nodejs/node/issues/39758
+      Promise.resolve().then(() => {
```

Comment:
I chose `Promise.resolve()` because it finishes before `setImmediate` (which finishes before `queueMicrotask`). Not sure what's best.

Reply:
I would prefer queueMicrotask I think